tests/test_plot_directive.py

```diff
@@ -0,0 +1,3 @@
+"""
+
+"""
```

Comment:
Is the blank test intended?

Reply:
This file should be deleted.  Plot directive is checked in test_tinypages.py